List freshly-added configuration properties in the README.md (#197)
hazelcast/hazelcast-hibernate: README.md

    
***NOTE:*** *QueryCache is always LOCAL to the member and never distributed across Hazelcast Cluster.*

#### Other Properties
- _cleanup_delay_ - the duration of physical local cache cleanups

> - _cluster_timeout_ - after how many milliseconds the client should stop retrying to establish a cluster connection

Reviewer: `the number of milliseconds the client should retry to establish a cluster connection`. maybe more clear?

Author: ```suggestion
- _cluster_timeout_ - the number of milliseconds the client should retry to establish a cluster connection
```

Author: works for me
